digdag-standards/src/main/java/io/digdag/standards/operator/td/TdWaitOperatorFactory.java

```diff
@@ -0,0 +1,132 @@
+package io.digdag.standards.operator.td;
+
+import com.google.inject.Inject;
+import com.treasuredata.client.TDClientHttpNotFoundException;
+import com.treasuredata.client.model.TDJobRequest;
+import com.treasuredata.client.model.TDJobRequestBuilder;
+import io.digdag.client.config.Config;
+import io.digdag.client.config.ConfigElement;
+import io.digdag.client.config.ConfigException;
+import io.digdag.spi.Operator;
+import io.digdag.spi.OperatorFactory;
+import io.digdag.spi.TaskExecutionException;
+import io.digdag.spi.TaskRequest;
+import io.digdag.spi.TaskResult;
+import io.digdag.spi.TemplateEngine;
+import io.digdag.util.BaseOperator;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.nio.file.Path;
+
+import static io.digdag.standards.operator.td.TdOperatorFactory.joinJob;
+import static java.nio.charset.StandardCharsets.UTF_8;
+
+public class TdWaitOperatorFactory
+        extends AbstractWaitOperatorFactory
+        implements OperatorFactory
+{
+    private static Logger logger = LoggerFactory.getLogger(TdWaitOperatorFactory.class);
+
+    private final TemplateEngine templateEngine;
+
+    @Inject
+    public TdWaitOperatorFactory(TemplateEngine templateEngine, Config systemConfig)
+    {
+        super(systemConfig);
+        this.templateEngine = templateEngine;
+    }
+
+    public String getType()
+    {
+        return "td_wait";
+    }
+
+    @Override
+    public Operator newTaskExecutor(Path workspacePath, TaskRequest request)
+    {
+        return new TdWaitOperator(workspacePath, request);
+    }
+
+    private class TdWaitOperator
+            extends BaseOperator
+    {
+        private TdWaitOperator(Path workspacePath, TaskRequest request)
+        {
+            super(workspacePath, request);
+        }
+
+        @Override
+        public TaskResult runTask()
+        {
+            Config params = request.getConfig().mergeDefault(
+                    request.getConfig().getNestedOrGetEmpty("td"));
+
+            int pollInterval = getPollInterval(params);
+
+            String query = templateEngine.templateCommand(workspacePath, params, "query", UTF_8);
+
+            int defaultRows = query.trim().isEmpty() ? 0 : 1;
```

Comment:
I was experimenting with having only one `td_wait` operator. This is a leftover from that. Good catch. Will fix. 👍